packages/renderer/src/lib/preferences/PreferencesConnectionCreationRendering.svelte

```diff
@@ -426,7 +426,12 @@ function closePage() {
           {#if connectionAuditResult && (connectionAuditResult.records?.length || 0) > 0}
             <AuditMessageBox auditResult="{connectionAuditResult}" />
           {/if}
-          <form novalidate class="p-2 space-y-7 h-fit" on:submit|preventDefault="{handleOnSubmit}" bind:this="{formEl}">
+          <form
+            novalidate
+            class="p-2 space-y-7 h-fit"
+            on:submit|preventDefault="{handleOnSubmit}"
+            bind:this="{formEl}"
+            aria-label="ResourceProperties">
```

Comment:
I think we can use space in the label name. Label should be more "human readable" than camelCase IMHO

"Properties information"

Reply:
that is a good point. For now we stick to camel case since I am not sure that AT can make a difference, or how it is read by such tool if there is space missing. 

I have checked existing aria-labels in the codebase, it is true that most cases are standard text with spaces. So let's stick to that variant if possible.

Reply:
It should be fixed, I'll keep it in mind for future labels.